toolbox/Xamarin.Forms.toolbox.xml

```diff
@@ -0,0 +1,167 @@
+			<Snippet>
+				<DatePicker />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Editor" Name="Editor" Image="view-Editor-16.png">
+			<Snippet>
+				<Editor />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Entry" Name="Entry" Image="view-Entry-16.png">
+			<Snippet>
+				<Entry Placeholder="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Image" Name="Image" Image="view-Image-16.png">
+			<Snippet>
+				<Image Source="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ListView" Name="ListView" Image="view-ListView-16.png">
+			<Snippet>
+				<ListView ItemsSource="">
+					<ListView.ItemTemplate>
+						<DataTemplate>
+						</DataTemplate>
+					</ListView.ItemTemplate>
+				</ListView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ProgressBar" Name="ProgressBar" Image="view-ProgressBar-16.png">
+			<Snippet>
+				<ProgressBar />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.SearchBar" Name="SearchBar" Image="view-SearchBar-16.png">
+			<Snippet>
+				<SearchBar Placeholder="" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Slider" Name="Slider" Image="view-Slider-16.png">
+			<Snippet>
+				<Slider Minimum="0" Maximum="1" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Stepper" Name="Stepper" Image="view-Stepper-16.png">
+			<Snippet>
+				<Stepper Minimum="0" Maximum="10" Increment="1" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Switch" Name="Switch" Image="view-Switch-16.png">
+			<Snippet>
+				<Switch IsToggled="false" />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.TableView" Name="TableView" Image="view-TableView-16.png">
+			<Snippet>
+				<TableView>
+					<TableRoot>
+						<TableSection>
+						</TableSection>
+					</TableRoot>
+				</TableView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.TimePicker" Name="TimePicker" Image="view-TimePicker-16.png">
+			<Snippet>
+				<TimePicker />
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.WebView" Name="WebViewWebView" Image="view-WebView-16.png">
+			<Snippet>
+				<WebView Source="" />
+			</Snippet>
+		</Item>
+	</Category>
+	<Category Name="Layouts">
+		<Item Class="Xamarin.Forms.ContentView" Name="ContentView" Image="layout-ContentView-16.png">
+			<Snippet>
+				<ContentView></ContentView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Frame" Name="Frame" Image="layout-Frame-16.png">
+			<Snippet>
+				<Frame></Frame>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.ScrollView" Name="ScrollView" Image="layout-ScrollView-16.png">
+			<Snippet>
+				<ScrollView></ScrollView>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.AbsoluteLayout" Name="AbsoluteLayout" Image="layout-AbsoluteLayout-16.png">
+			<Snippet>
+				<AbsoluteLayout></AbsoluteLayout>
+			</Snippet>
+		</Item>
+		<Item Class="Xamarin.Forms.Grid" Name="Grid" Image="layout-Grid-16.png">
```

Comment:
you forgot the `F***Layout`

Reply:
Not in 15-5 so yeah we dont include it here